internal/pkg/cli/svc_deploy.go

```diff
@@ -108,7 +109,8 @@ type deploySvcOpts struct {
 	buildRequired     bool
 	appEnvResources   *stack.AppRegionalResources
 	rdSvcAlias        string
-	svcUpdater        serviceUpdater
+	svcUpdater        svcForceUpdater
+	now               func() time.Time
```

Comment:
I think we need to update `init.go` and `deploy.go` as well with `now: time.Now`

Reply:
Oooh good call!